### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by padu12**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/db?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=1234
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.datasource.url=jdbc:mysql://localhost:3306/db?serverTimezone=UTC
+#spring.datasource.username=root
+#spring.datasource.password=1234
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```
Comment (Den-moroz) [inline src/main/resources/application.properties]: u can delete it, or do u need it?

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ports:
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url" : "jdbc:postgresql://db:$POSTGRES_DOCKER_PORT/$POSTGRES_DATABASE",
+        "spring.datasource.username" : "$POSTGRES_USER",
+        "spring.datasource.password" : "$POSTGRES_PASSWORD",
+        "spring.jpa.properties.hibernate.dialect" : "org.hibernate.dialect.PostgreSQLDialect"
+      }'
```
Comment (Den-moroz) [inline docker-compose.yaml]: add empty line

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Builder stage
+FROM openjdk:17-jdk-slim as builder
+WORKDIR application
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+# Final stage
```
Comment (Den-moroz) [inline Dockerfile]: remove comment

**Review by Den-moroz — APPROVED** [review]
Good job! Let's fix some minor moments)

File: src/main/resources/application.properties (hunk 1 above)
Comment (padu12) [reply src/main/resources/application.properties]: no, I don't)

**Review by andrii-hoienko — APPROVED** [review]
Nice

**Pull request merged**